src/eradiate/contexts.py

```diff
@@ -305,6 +305,15 @@ class KernelDictContext(Context):
         default="True",
     )
 
+    atmosphere_medium_id: t.Optional[str] = documented(
+        attr.ib(default=None, converter=attr.converters.optional(str)),
+        doc="If relevant, value which must be used as the surrounding medium for sensors, which"
+        "are positioned inside the atmosphere volume.",
```

Comment:
Let's also broaden the scope of this doc:
```suggestion
        doc="If relevant, a kernel medium ID to reference by other kernel components.",
```